### PR TITLE
Update History pushstate|replacestate note about “unused” param (was “title”)

### PR DESCRIPTION
**Pull request by sideshowbarker**
Related MDN change: https://github.com/mdn/content/pull/13322 • Related spec bug: https://github.com/whatwg/html/pull/6482

Relevant spec sections:

- https://html.spec.whatwg.org/multipage/history.html#dom-history-pushstate
- https://html.spec.whatwg.org/multipage/history.html#dom-history-replacestate
- https://html.spec.whatwg.org/multipage/history.html#dom-history-pushstate-dev
- https://html.spec.whatwg.org/multipage/history.html#dom-history-replacestate-dev

### CODE REVIEW

File: api/History.json [hunk 1]
```diff
@@ -296,7 +296,7 @@
         },
         "title_parameter": {
```
Comment (ddbeck) [inline api/History.json:297]: Is there a reason you didn't rename the features themselves?

Comment (sideshowbarker) [inline api/History.json:297]:
```suggestion
        "unused_parameter": {
```

File: api/History.json [hunk 2]
```diff
@@ -397,7 +397,7 @@
...
```
Comment (sideshowbarker) [inline api/History.json:398]:
```suggestion
        "unusued_parameter": {
```

File: api/History.json (hunk 1 above)
Comment (sideshowbarker) [reply api/History.json]: > Is there a reason you didn't rename the features themselves?

Nope, it was just an oversight/miss — thanks for catching that

**Pull request merged**